fast/stages-multitenant/0-bootstrap-tenant/cicd.tf

```diff
@@ -29,11 +29,12 @@ locals {
     {
       for k, v in google_iam_workload_identity_pool_provider.default :
       k => {
-        issuer           = local.identity_providers[k].issuer
-        issuer_uri       = local.identity_providers[k].issuer_uri
-        name             = v.name
-        principal_tpl    = local.identity_providers[k].principal_tpl
-        principalset_tpl = local.identity_providers[k].principalset_tpl
+        allowed_audiences = v.oidc.0.allowed_audiences
```

Comment:
did you try this with other providers?

Reply:
Yes, it doesn't have any impact because it's only adding a new property to the local variable.